src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptionsExtensions.cs

```diff
@@ -89,13 +89,55 @@ public static THeaders GetHeaders<THeaders>(this OtlpExporterOptions options, Ac
     }
 
     public static OtlpExporterTransmissionHandler<TraceOtlpCollector.ExportTraceServiceRequest> GetTraceExportTransmissionHandler(this OtlpExporterOptions options)
-        => new(GetTraceExportClient(options));
+    {
+        var exportClient = GetTraceExportClient(options);
+
+        double timeoutMilliseconds = options.TimeoutMilliseconds;
```

Comment:
nit: It might be more readable to use the conditional operator here. Just my personal opinion though 😀

```csharp
double timeoutMilliseconds = exportClient is OtlpHttpTraceExportClient httpTraceExportClient
    ? httpTraceExportClient.HttpClient.Timeout.TotalMilliseconds
    : options.TimeoutMilliseconds;
```